packages/editor-ui/src/components/ExpressionParameterInput.vue

```diff
@@ -0,0 +1,297 @@
+<template>
+	<div :class="$style['expression-parameter-input']" v-click-outside="onBlur">
+		<div :class="[$style['all-sections'], { [$style['focused']]: isFocused }]">
+			<div
+				:class="[
+					$style['prepend-section'],
+					'el-input-group__prepend',
+					{ [$style['squared']]: isForRecordLocator },
+				]"
+			>
+				<ExpressionFunctionIcon />
+			</div>
+			<InlineExpressionEditorInput
```

Comment:
reminder to switch to kebab case

Reply:
That is what I wanted and [argued for](https://www.notion.so/n8n/Component-Guidelines-8b323d1db6ec44fb9d296dcaa589465c#29d5258426874b3c9d2d00ea17109677) and was overruled on because guidelines say the opposite.

<img width="676" alt="image" src="https://user-images.githubusercontent.com/44588767/207270364-7c27a3cf-724f-4ccc-abd1-4e04973e7a5e.png">

Reply:
ops my bad sorry.. this is good :)